packages/connect/src/device/DeviceList.ts

```diff
@@ -62,27 +61,24 @@ export class DeviceList extends TypedEmitter<DeviceListEvents> {
 
     transportFirstEventPromise: Promise<void> | undefined;
 
-    constructor() {
+    constructor({
+        transports = ['BridgeTransport'],
```

Comment:
This fallback won't work anymore if you pass an empty array. 
But I'm not sure if that can really happen at this point, maybe it's not important

Reply:
ha, good point, I ll put it back. didn't want to make any runtime changes like this one